cli/Valet/PhpFpm.php

```diff
@@ -192,8 +253,31 @@ public function useVersion($version, $force = false)
             $this->brew->ensureInstalled($version, [], $this->taps);
         }
 
-        // Unlink the current php if there is one
+        // Delete old Valet sock files, install the new version, and, if this is a global change, unlink and link PHP
+        if ($directory && $site) {
```

Comment:
I don't think it's possible to reach here without `$site` being set *if* `$directory` is set, so I think we can simplify this conditional, right?

```suggestion
        if ($directory) {
```